app/controllers/pet_applications_controller.rb

```diff
@@ -0,0 +1,32 @@
+class PetApplicationsController < ApplicationController
+  def new
+  end
+
+  def create
+    @application = PetApplication.create(name: params[:name], street: params[:street], city: params[:city], state: params[:state], zip: params[:zip], description: params[:description])
```

Comment:
Instead of listing out all the attributes for creating a model object, you want to make use of strong params.